README.md

```diff
@@ -11,8 +11,7 @@
 bevy gaussian splatting render pipeline plugin. view the [live demo](https://mosure.github.io/bevy_gaussian_splatting/index.html?arg1=cactus.gcloud)
 
 ![Alt text](docs/notferris.png)
-![Alt text](docs/cactus.gif)
-![Alt text](docs/bike.png)
```

Comment:
just curious, is this primarily to 'declutter' the README?

Reply:
a bit of declutter and also showcasing the latest results. `cactus.gif`/`bike.png` were recorded with incorrect sRGB colorspace, flickering radix sort, and un-normalized quaternion errors.